src/devtools/client/inspector/rules/rules.js

```diff
@@ -346,8 +346,8 @@ class RulesView {
    *         The pseudo class to toggle on or off.
    */
   onTogglePseudoClass(value) {
-    this.store.dispatch(togglePseudoClass(value));
-    this.inspector.togglePseudoClass(value);
+    // this.store.dispatch(togglePseudoClass(value));
+    // this.inspector.togglePseudoClass(value);
   }
```

Comment:
lets delete more code paths like this that will never be used. Everytime we touch the inspector we should deleting lots of code :)

Reply:
I meant to save it for later as it may someone some time, but sure, I can remove code. :)

Reply:
Understood. We can always look at how Firefox works. We should prioritize what we're currently supporting